ztp/gitops-subscriptions/argocd/resource-hook-example/siteconfig/test-3node.yaml

```diff
@@ -0,0 +1,155 @@
+# test-node[123]-bmh-secret & assisted-deployment-pull-secret need to be created under same namespace test-3node
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-3node
+  labels:
+    name: test-3node
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "test-3node"
+  namespace: "test-3node"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "ssh-rsa AAAA..."
+  clusters:
+  - clusterName: "test-3node"
+    clusterType: "standard"
+    clusterProfile: "du"
+    numMasters: 3
+    clusterLabels:
+      group-du-sno: ""
+      common: true
+      sites : "test-3node"
+    clusterNetwork:
+      - cidr: 1001:db9::/48
+        hostPrefix: 64
+    machineNetwork:
+      - cidr: 2620:52:0:10e7::/64
```

Comment:
Maybe use clearly fictitious prefixes here to indicate we expect them to be changed for real clusters? 1111:2222:3333:4444::/64? Same below for bmc and node IPs.